File: samples/CS/03-MenuDropDown/Form1.cs
using System;
using System.Windows.Forms;
using WinForms.Ribbon;

namespace _03_MenuDropDown
{
    public partial class Form1 : Form
    {
        private RibbonItems _ribbonItems;

        public Form1()
        {
            InitializeComponent();
            _ribbonItems = new RibbonItems(_ribbon);
            _ribbonItems.Init();

        }
    }
}

File: samples/VB/03-MenuDropDown/Form1.vb
Imports WinForms.Ribbon

Namespace _03_MenuDropDown
    Partial Public Class Form1
        Inherits Form
        Private _ribbonItems As RibbonItems

        Public Sub New()
            InitializeComponent()
            _ribbonItems = New RibbonItems(_ribbon)
            _ribbonItems.Init()
        End Sub
    End Class
End Namespace
